.github/workflows/ci.yml

```diff
@@ -5,28 +5,297 @@ on:
   release:
     types: [published]
 
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME_TESTING: ${{ github.repository }}/testing
+  IMAGE_NAME_RUNTIME: ${{ github.repository }}/runtime
+
 jobs:
-  ci:
+  test:
+    name: "Test backpex"
     runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
+    strategy:
+      matrix:
+        elixir: ['1.15.7', '1.16.1']
+        erlang: ['26.1.2', '26.2.1']
```

Comment:
Feel free to add more version.

Reply:
The latest two OTP versions (25 and 26) should be fine, I think.